rpc/backend/backend.go

```diff
@@ -134,6 +125,7 @@ var _ BackendI = (*Backend)(nil)
 type Backend struct {
 	ctx                 context.Context
 	clientCtx           client.Context
+	rpcClient           tmrpcclient.Client
```

Comment:
we're using only methods from the `SignClient` interface
```suggestion
	rpcClient           tmrpcclient.SignClient
```